auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/UmsClient.java

```diff
@@ -451,6 +453,23 @@ public Account getAccount(String requestId, String userCrn) {
         ).getAccount();
     }
 
+    /**
+     * Wraps a call to getActorWorkloadCredentials.
+     *
+     * @param requestId the request ID for the request
+     * @param userCrn   the user CRN
+     * @return the ActorWorkloadCredentialsResponse
+     */
+    public GetActorWorkloadCredentialsResponse getActorWorkloadCredentials(String requestId, String userCrn) {
+        checkNotNull(requestId);
+        checkNotNull(userCrn);
+        return newStub(requestId).getActorWorkloadCredentials(
+            GetActorWorkloadCredentialsRequest.newBuilder()
+                .setActorCrn(Crn.fromString(userCrn).getAccountId())
```

Comment:
Should be possible to use the userCrn directly - assuming it's actually a crn.

Reply:
Done. Thanks.